ptpython/config.py

```diff
@@ -0,0 +1,129 @@
+"""
+Ptpython settings loader from config file.
+"""
+import copy
+import os
+import re
+import six
+from six.moves.configparser import ConfigParser
+
+
+option_re = re.compile('^(?P<option>\w+)\s+=.+(?P<comment>#.*)?$')
+
+
+class Settings:
+    """
+    Ptpython user defined settings, loaded from <config-dir>/conf.cfg file.
+
+    :param defaults: dictionary with repl's settings customisable in conf.cfg.
+    """
+    user_defined = {}
```

Comment:
This is not an instance variable. A second instance of ``Settings`` will share the same ``user_defined`` attribute.